Name: Reset Options On Power Button Hold
Version: 1.3.1-1
Author: Jason Robitaille
Description: Goes straight to the reset options, skipping the airplaine mode part, on power button hold  (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -293,7 +293,7 @@
 	
 	if(!this.deviceLocked) {
 		var appController = Mojo.Controller.getAppController();
-		var stageController = appController.getStageProxy("PowerOffAlert");
+		var stageController = appController.getStageProxy("ResetOptionAlert");
 		if(stageController)
 			stageController.delegateToSceneAssistant("cancelTimer");
 	}
@@ -2247,7 +2247,7 @@
 showPowerOffAlert: function() {
 	
 	var appController = Mojo.Controller.getAppController();
-	var stageController = appController.getStageProxy("PowerOffAlert");
+	var stageController = appController.getStageProxy("ResetOptionAlert");
         // if the window already exists, don't do anything
         if (!stageController) {
          	// Create a popuo alert
@@ -2252,7 +2252,7 @@
         if (!stageController) {
          	// Create a popuo alert
 		    var f = function(stageController){
-		      stageController.pushScene('poweroffalert',this);
+		      stageController.pushScene('resetalert',this);
 		    }.bind(this);
 		    var params = {name: 'PowerOffAlert', height: 175,lightweight:true};		    	    
 		    appController.createStageWithCallback(params,f,'popupalert');
@@ -2254,7 +2254,7 @@
 		    var f = function(stageController){
 		      stageController.pushScene('poweroffalert',this);
 		    }.bind(this);
-		    var params = {name: 'PowerOffAlert', height: 175,lightweight:true};		    	    
+		    var params = {name: 'ResetOptionAlert', height: 175,lightweight:true};		    	    
 		    appController.createStageWithCallback(params,f,'popupalert');
         }	
 },
